pyaedt/edb_core/siwave.py

```diff
@@ -1273,3 +1277,109 @@ def create_pin_group_terminal(self, source):
         else:
             pass
         return pos_pingroup_terminal.GetName()
+
+    @pyaedt_function_handler()
+    def configure_siw_analysis_setup(self, simulation_setup=None):
+        """Configure Siwave analysis setup.
+        Parameters
```

Comment:
I think a blank line needs to be added between the description and "Parameters" heading for proper rendering in the documenation.